### PR TITLE
Firebase Auth - upgrade iOS deployment targets

### PR DESCRIPTION
**Pull request by ericwindmill**
Upgrades Firebase plugins and minimum iOS deployment targets

### CODE REVIEW

File: firebase-auth-flutterfire-ui/complete/ios/Podfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
```
Comment (domesticmouse) [inline firebase-auth-flutterfire-ui/complete/ios/Podfile:1]: Can you change this line to refer to the documentation about why 13 is the required minimum.

**Review by domesticmouse — COMMENTED** [review]
We are also going to have to update the `codelab_rebuild.yaml` to reflect these changes so it can be updated each quarter.

File: firebase-auth-flutterfire-ui/codelab_rebuild.yaml [hunk 2]
```diff
@@ -328,6 +328,54 @@ steps:
           	<string>1:963656261848:ios:d9e01cfe8b675dfcb237ad</string>
           </dict>
           </plist>
+      - name: ios/Podfile
```
Comment (ericwindmill) [inline firebase-auth-flutterfire-ui/codelab_rebuild.yaml:331]: @domesticmouse I'm not sure that this is correct, and I think this only solves the problem for the "complete" flutter project, and not the 'start' project. In this file, 100% of the 'path' arguments point to "complete/....", which has me confused. 

Do I just need to do the same thing with the path argument "start/ios/Podfile"?

**Comment by domesticmouse** [comment]
I'll figure it out

File: firebase-auth-flutterfire-ui/codelab_rebuild.yaml (hunk 2 above)
Comment (domesticmouse) [reply firebase-auth-flutterfire-ui/codelab_rebuild.yaml:331]: I can rebuild the `codelab_rebuild.yaml` file to respect the `complete` and `start` steps. I think I possibly did something mind bending when I created this rebuild file.

**Comment by domesticmouse** [comment]
I've attempted to rebuild the `codelab_rebuild.yaml` for Flutter 3.24. Can you check whether I goofed and broke something?

**Comment by ericwindmill** [comment]
> I've attempted to rebuild the `codelab_rebuild.yaml` for Flutter 3.24. Can you check whether I goofed and broke something?

Everything is working and looks good on my end

**Comment by domesticmouse** [comment]
Closing this PR, as I've merged it into https://github.com/flutter/codelabs/pull/2220 for landing with the release.

**Pull request closed without merging**